acr-build.yml

```diff
@@ -2,34 +2,61 @@ version: v1.1.0
 steps:
   - id: ccd
     build: >
-      -t {{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-latest -t {{.Run.Registry}}/ccd/logstash:latest
+      -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-latest -t {{.Run.Registry}}/ccd/logstash:latest
       --build-arg conf=ccd_logstash.conf.in
       .
     keep: true
 
   - id: probate
     build: >
-      -t {{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:probate-latest
+      -t {{.Run.Registry}}/ccd/logstash:ccd-probate-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-probate-logstash-latest
       --build-arg conf=ccdprobate_logstash.conf.in
```

Comment:
we might want a similar config for the Probate own instance of LS used to index legacy cases from a Probate own DB, as configured in `probate_logstash.conf.in`
This is  different from the CCD LS that indexes CCD Probate cases from the CCD DB, configured in `ccdprobate_logstash.conf.in`